TEMPLATE.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 000-template
```

Comment:
Would it be helpful to provide details of the numbering and where it comes from? I've seen some confusion as to how/where derive this from

Reply:
Would be nice to run the template through a markdown linter (e.g. empty lines around lists), to make sure that future issues are created with a sane format.

Reply:
The only one I'm aware of is using Visual Studio Code with the [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) extension. Should work well for semi-technical people.